src/viewer.js

```diff
@@ -480,22 +479,16 @@ class VolumeImageViewer {
    */
   constructor (options) {
     this[_options] = options
-    if ('useWebGL' in this[_options]) {
-      this[_usewebgl] = this[_options].useWebGL
-    } else {
-      this[_usewebgl] = true
+    if (!('useWebGL' in this[_options])) {
+      this[_options].useWebGL = true
```

Comment:
This doesn't account for the option `{ useWebGL: false }`

Reply:
Or in other words, the option may be `undefined` rather than `false` which is not ideal in my opinion.

Reply:
no if at in the options of the constructor is passed, after the check it will be:
1) { useWebGL: false }, this[_options].useWebGL will be false
2) { useWebGL: true }, this[_options].useWebGL will be true
3) {  } (i.e. undefined), this[_options].useWebGL will be true (i.e. the default)